azure-quantum/requirements-qiskit.txt

```diff
@@ -1,6 +1,5 @@
 qiskit-ionq>=0.3.3,<1.0
-# TODO(kuzminrobin, #494): See line below. Upon fix of #494 consider reverting fragment `<0.25.0` back to `<1.0`.
-qiskit-terra>=0.19.1,<0.25.0
 qiskit-qir>=0.4,<0.5
+qiskit>=0.45,<0.46
```

Comment:
Do we need to keep this as qiskit-terra for now until qiskit-ionq and qiskit-qir makes their swap? Otherwise, we could be pulling in conflicting package from those other dependencies.

Reply:
Reverted to use `qiskit-terra`. Thanks!